ovirt/CentOS-7-x86_64-oVirt-r1.ks

```diff
@@ -0,0 +1,180 @@
+auth --enableshadow --passalgo=sha512
+reboot
+url --url="mirror.centos.org/centos/7/os/x86_64"
+firewall --enabled --service=ssh
+firstboot --disable
+ignoredisk --only-use=vda
+keyboard --vckeymap=us --xlayouts='us'
+# System language
+lang en_US.UTF-8
+repo --name "os" --baseurl="http://mirror.centos.org/centos/7/os/x86_64/" --cost=100
+repo --name "updates" --baseurl="http://mirror.centos.org/centos/7/updates/x86_64/" --cost=100
+repo --name "extras" --baseurl="http://mirror.centos.org/centos/7/extras/x86_64/" --cost=100
+
+# oVirt specific repos
+repo --name "centos-ovirt42" --baseurl="http://mirror.centos.org/centos/7/virt/x86_64/ovirt-4.2/" --cost=100
+
+# Network information
+network  --bootproto=dhcp
+network  --hostname=localhost.localdomain
+# Root password
+rootpw --iscrypted thereisnopasswordanditslocked
```

Comment:
why not just lock the rootpw ?

Reply:
This is exactly what there is in the generic cloud image, I just added the ovirt repo and the agent package.